admin/client/client.go

```diff
@@ -30,7 +30,8 @@ const retryPolicy = `{"methodConfig": [{
 type Client struct {
 	adminv1.AdminServiceClient
 	adminv1.AIServiceClient
-	conn *grpc.ClientConn
+	Token string
```

Comment:
exposing just for tests but I guess no other way around this

Reply:
Yeah, but could conceivably be useful for non-test use cases also in the future I guess.